docs/backend-system/building-backends/08-migrating.md

```diff
@@ -1063,3 +1063,291 @@ backend.start();
+/* highlight-add-end */
+```
+
+##### TechDocs
+
+A basic installation of the Search plugin with the TechDocs collator will look as follows:
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+
+// Other plugins...
+
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-search-backend/alpha'));
+backend.add(import('@backstage/plugin-search-backend-module-techdocs/alpha'));
+/* highlight-add-end */
+```
+
+### The Permission Plugin
+
+A basic installation of the Permission plugin will look as follows:
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+
+// Other plugins...
+
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-permission-backend/alpha'));
+backend.add(
+  import('@backstage/plugin-permission-backend-module-allow-all-policy'),
+);
+/* highlight-add-end */
+```
+
+> Note: The above example includes a default allow-all policy. If that is not what you want, do not add the second line and instead investigate one of the options below.
+
+#### Custom Permission Policy
+
+In order to add your own permission policy you'll need to do the following:
+
+```ts
+import { createBackendModule } from '@backstage/backend-plugin-api';
+import { BackstageIdentityResponse } from '@backstage/plugin-auth-node';
+import {
+  PolicyDecision,
+  AuthorizeResult,
+} from '@backstage/plugin-permission-common';
+import {
+  PermissionPolicy,
+  PolicyQuery,
+} from '@backstage/plugin-permission-node';
+import { policyExtensionPoint } from '@backstage/plugin-permission-node/alpha';
+
+class CustomPermissionPolicy implements PermissionPolicy {
+  async handle(
+    request: PolicyQuery,
+    user?: BackstageIdentityResponse,
+  ): Promise<PolicyDecision> {
+    // TODO: Add code here that inspects the incoming request and user, and returns AuthorizeResult.ALLOW, AuthorizeResult.DENY, or AuthorizeResult.CONDITIONAL as needed. See the docs at https://backstage.io/docs/permissions/writing-a-policy for more information
+
+    return {
+      result: AuthorizeResult.ALLOW,
+    };
+  }
+}
+
+const customPermissionBackendModule = createBackendModule({
+  pluginId: 'permission',
+  moduleId: 'custom-policy',
+  register(reg) {
+    reg.registerInit({
+      deps: { policy: policyExtensionPoint },
+      async init({ policy }) {
+        policy.setPolicy(new CustomPermissionPolicy());
+      },
+    });
+  },
+});
+
+const backend = createBackend();
+
+// Other plugins...
+
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-permission-backend/alpha'));
+backend.add(customPermissionBackendModule);
+/* highlight-add-end */
+```
+
+### The TechDocs Plugin
+
+A basic installation of the TechDocs plugin will look as follows:
+
+```ts title="packages/backend/src/index.ts"
+const backend = createBackend();
+
+// Other plugins...
+
+/* highlight-add-start */
+backend.add(import('@backstage/plugin-search-backend-module-techdocs/alpha'));
```

Comment:
I see you addressed my comment before I corrected my url recommendation. 😊 Sorry, that was the search techdocs url. The main techdocs one is: `@backstage/plugin-techdocs-backend/alpha`.

Reply:
You are absolutely correct, trying to move too fast! Really want this update merged as I know it will help. many Adopters but it needs to be right so really thankful you followed up!!!

Reply:
Also, this has been fixed 👍